packages/webpack/test/fixtures/main/node_modules/prototype-poisoning-package/tostring.js

```diff
@@ -0,0 +1,12 @@
+// poisoning the specifier with a toString
+
+let count = 0;
+require('path') // this is here for policy generator to find
+const stolenFs = __webpack_require__({toString: () => {
+  count++;
+  if (count < 3) {
```

Comment:
why `3` btw?

Reply:
first 2 times that field is being inspected we want to pretend we're importing `path` so that it ends up validating against the policy and then we switch to something more powerful.